core/services/workflows/syncer/fetcher.go

```diff
@@ -93,6 +93,10 @@ func (s *FetcherService) Fetch(ctx context.Context, url string) ([]byte, error)
 		return nil, err
 	}
 
+	if err := resp.Validate(); err != nil {
```

Comment:
@MStreet3 I'm not sure this is validating the right thing -- the audit feedback was saying we should validate the underlying message, i.e. ghcapabilities.Response, not api.Message